openlcs/products/views.py

```diff
@@ -89,6 +139,7 @@ def list(self, request, *args, **kwargs):
         """
         return super().list(request, *args, **kwargs)
 
+    @cache_response(timeout=60 * 15, key_func=ReleaseObjectKeyConstructor())
```

Comment:
Could you add this timeout setting in the `settings.py`

Reply:
I mean defined this override timeout to setting, so that , we only need to change the setting.py

Reply:
sure, DONE